tests/test_blocking.py

```diff
@@ -3,10 +3,13 @@
 import random
 import unittest
 
+import bitarray
+import pytest
```

Comment:
In addition to adding tests, I also modified this file to use Pytest instead of Unittest.